frontend/src/container/GroupedNotificationItem.jsx

```diff
@@ -9,200 +9,196 @@ import {
+            <div className='notification__list__item__text__avatars'>
+              <Avatar
+                size={numberOfAuthors === 2 ? AVATAR_SIZE.MINI : AVATAR_SIZE.SMALL}
+                apiUrl={FETCH_CONFIG.apiUrl}
+                user={notification.author[0]}
+              />
+              {numberOfAuthors === 2 && (
                 <Avatar
-                  size={numberOfAuthors === 2 ? AVATAR_SIZE.MINI : AVATAR_SIZE.SMALL}
+                  size={AVATAR_SIZE.MINI}
                   apiUrl={FETCH_CONFIG.apiUrl}
-                  user={notification.author[0]}
+                  user={notification.author[1]}
+                  customClass='notification__list__item__text__avatars__second'
                 />
-                {numberOfAuthors === 2 && (
-                  <Avatar
-                    size={AVATAR_SIZE.MINI}
-                    apiUrl={FETCH_CONFIG.apiUrl}
-                    user={notification.author[1]}
-                    customClass='notification__list__item__text__avatars__second'
-                  />
-                )}
-              </div>
-            ) : (
-              <Icon
-                customClass='notification__list__item__text__usersAvatar'
-                icon='fas fa-fw fa-users'
-                color='#fdfdfd' // INFO - GB - 2021-08-26 - offWhite color
-                title={notification.author.map(author => author.publicName)}
-              />
-            )}
-          <span
-            className='notification__list__item__text__content'
-            dangerouslySetInnerHTML={{
-              __html: (
-                notificationDetails.text + ' ' +
-                `<span title='${escapeHtml(formatAbsoluteDate(notification.created, user.lang))}'\\>`
-              )
-            }}
-          />
-        </div>
-        <div className='notification__list__item__meta'>
-          <div
-            className='notification__list__item__meta__date'
-            title={formatAbsoluteDate(notification.created, user.lang)}
-          >
-            {props.shortDate(notification.created)}
-          </div>
-          <div className='notification__list__item__meta__space'>
-            {(numberOfWorkspaces === 1 && notification.group[0].workspace &&
-              notification.group[0].workspace.label
-            )}
-          </div>
+              )}
+            </div>
+          ) : (
+            <Icon
+              customClass='notification__list__item__text__usersAvatar'
+              icon='fas fa-fw fa-users'
+              color='#fdfdfd' // INFO - GB - 2021-08-26 - offWhite color
+              title={notification.author.map(author => author.publicName)}
+            />
+          )}
+        <span
+          className='notification__list__item__text__content'
+          dangerouslySetInnerHTML={{
+            __html: (
+              notificationDetails.text + ' ' +
+              `<span title='${escapeHtml(formatAbsoluteDate(notification.created, user.lang))}'\\>`
```

Comment:
I know it's not you but I wonder why there is a \\ before > on this span.

I'm expecting to have some content in this span and span is not self-closing. Moreover, a self-closing element would be closed by /, not \.